index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Deadpool Connect 4</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="main.js"></script>
+</head>
+<body>
+
+<header class="header1">
+    <img class="headerImage" src="DPsprawl.png">
+</header>
+<main>
+    <div class="playerBoxBorder">
+        <div class="playerBox1 playerBox">
+            <div class="pieceDisplay1">
```

Comment:
So in reality you did this as player1

you probably want to do your css as nested
```
<div class="player1 playerBox">
    <div class="pieceDisplay">
```
and in css
```
.player1 > .pieceDisplay
```